docs/source/deployguide/cadeploy.md

```diff
@@ -0,0 +1,597 @@
+
+### Delete the TLS CA server certificates
+
+Before starting the server, if you modified the `csr` values in the configuration .yaml file, you need to delete the `fabric-ca-server-tls/ca-cert.pem` file and the entire `fabric-ca-server-tls/msp` folder. These certificates will be re-generated when you start the CA server in the next step.
+
+### Start the TLS CA server
+
+Run the following command to start the CA server:
+```
+./fabric-ca-server start
+```
+When the server starts successfully you will see something similar to:
+```
+[INFO] Listening on https://0.0.0.0:7054
+```
+Because you have enabled TLS communications, notice that the TLS signed certificate `tls-cert.pem` file is generated under the `FABRIC_CA_HOME` location.
+
+**Tip:** The CA `ADMIN_USER` and `ADMIN_PWD` that were set on the `init` command cannot be overridden with the `-b` flag on this `start` command. When you need to modify the CA admin password, use the Fabric CA client [identity](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/clientcli.html#identity-command) command.
+
+**Optional flags**:
+- `-d` - If you want to run the server in DEBUG mode which facilitates problem diagnosis, you can include the `-d` flag on the start command. However, in general it is not recommended to run a server with debug enabled as this will cause the server to perform slower.
+- `-p` - If you want the server to run on a port different than what is specified in the configuration .yaml file, you can override the existing port.
+
+### Enroll bootstrap user with TLS CA
+
+Now that your TLS CA is configured and before you can deploy any other nodes for your organization, you need to enroll the bootstrap user of the node to generate the TLS certificates. Since the CA server is up and running, instead of using the **Fabric CA server CLI commands** we now use the **Fabric CA client CLI commands** to submit an enrollment request to the server. Before you can deploy another new CA with TLS enabled, you need to provide the new CA TLS certificate and private key to the configuration which allows the CA to transact securely with the other nodes in the network. The following process is described for a CA node, but can be easily extended for peer or ordering nodes.
+
+Performed by using the Fabric CA client, the enrollment process is used to generate the certificate and private key pair which forms the node TLS identity. You should have already setup the required folders in the [Fabric CA client](#fabric-ca-client) section.
+
+The folder structure we are using for these commands is
+  ```
+  fabric-ca-client
+    └── tls-ca
+      ├── admin
+      └── tls
+  ```
+  These folders are used by the Fabric CA client to:
+   - Store the certificates that are issued when the Fabric CA client enroll command is run against the TLS CA server. (**admin** folder)
```

Comment:
In the CA client section above, it says certificates from the enroll command will go to FABRIC_CA_CLIENT_HOME/msp directory.
Is the admin folder just for enrollments of admins?

Reply:
Based on your other comment, I've removed all of the msp sub-folders from the /admins directory so this is now more precise:
`Store the certificates that are issued when the Fabric CA client enroll command is run against the TLS CA server to enroll the TLS CA bootstrap identity. (**admin** folder)`